frontend/type/Motherboard.ts

```diff
@@ -0,0 +1,8 @@
+export default interface Motherboard {
+    _id: string,
+    ram_slots: 4,
```

Comment:
why just 4?

Reply:
@IntiveMichal no idea... I replaced the `4` with `number`.